README.md

```diff
@@ -327,11 +328,11 @@ Sometimes you'll want to modify the failure message in some way. You can do
   method: 'get',
   url: 'https://{{dc}}.api.mailchimp.com/2.0/users?apikey={{apiKey}}',
   expects: 200,
-  afterFailure: function (err, params) {
+  afterFailure: function (err, params, res) {
     if (err.response.statusCode === 403) {
       err.code = 'oauth_refresh';
```

Comment:
I know that `oauth_refresh` still works. But it might be best to stay consistent with our current `#oauth_refresh`

Reply:
Ah yh, good catch